core/src/main/java/hudson/model/Slave.java

```diff
@@ -529,7 +529,12 @@ private void reportLauncherCreateError(@Nonnull String humanReadableMsg, @CheckF
 
     /**
      * Gets the corresponding computer object.
+     *
+     * @return
```

Comment:
Even though people believe it is not possible for a Slave to have zero executors this is not true, as the in memory model will be valid whilst it is being removed.
This was also copied from `Node.toComputer()`